extensions/wikia/Email/templates/temporaryPassword.mustache

```diff
@@ -3,8 +3,11 @@
 	<tr>
 		<td class="widest">
 			<p>{{passwordIntro}}</p>
+			{{# username}}
+				<p><pre>{{username}}</pre></p>
```

Comment:
Well, the designs don't really go to that level of detail so I just added <pre>'s for the password to help it stand out.  I don't really have a strong opinion about it.